packages/attribute-registry-process/test/db.test.ts

```diff
@@ -163,81 +163,217 @@ describe("database test", () => {
     });
 
     describe("readModelService", () => {
+      let attribute1: Attribute;
+      let attribute2: Attribute;
+      let attribute3: Attribute;
+      let attribute4: Attribute;
+      let attribute5: Attribute;
+      let attribute6: Attribute;
+      let attribute7: Attribute;
+
+      beforeEach(async () => {
+        attribute1 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 001 test",
+          kind: attributeKind.certified,
+          origin: "IPA",
+          code: "12345A",
+        };
+        await addOneAttribute(attribute1);
+
+        attribute2 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 002 test",
+          kind: attributeKind.certified,
+          origin: "IPA",
+          code: "12345B",
+        };
+        await addOneAttribute(attribute2);
+
+        attribute3 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 003 test",
+          kind: attributeKind.certified,
+          origin: "IPA",
+          code: "12345C",
+        };
+        await addOneAttribute(attribute3);
+
+        attribute4 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 004",
+          kind: attributeKind.declared,
+        };
+        await addOneAttribute(attribute4);
+
+        attribute5 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 005",
+          kind: attributeKind.declared,
+        };
+        await addOneAttribute(attribute5);
+
+        attribute6 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 006",
+          kind: attributeKind.verified,
+        };
+        await addOneAttribute(attribute6);
+
+        attribute7 = {
+          ...mockAttribute,
+          id: uuidv4(),
+          name: "attribute 007",
+          kind: attributeKind.verified,
+        };
+        await addOneAttribute(attribute7);
+      });
```

Comment:
```suggestion
```
I wouldn't do this here because when we are going to add the test for the POSTs we will have a dirty db with already these attributes inside.

Since this preparation is common across several tests, I would move it in a dedicated method which will be called from those who need it.